src/Commands/SettingsCommandGroup.cs

```
@@ -80,23 +81,40 @@ public class SettingsCommandGroup : CommandGroup {
         var cfg = await _dataService.GetSettings(guildId, CancellationToken);
         Messages.Culture = GuildSettings.Language.Get(cfg);
 
-        return await SendSettingsListAsync(cfg, currentUser, CancellationToken);
+        return await SendSettingsListAsync(cfg, currentUser, page, CancellationToken);
     }
 
-    private async Task<Result> SendSettingsListAsync(JsonNode cfg, IUser currentUser, CancellationToken ct = default) {
+    private async Task<Result> SendSettingsListAsync(JsonNode cfg, IUser currentUser, int page, CancellationToken ct = default) {
         var builder = new StringBuilder();
-
-        foreach (var option in AllOptions) {
-            builder.Append(Markdown.InlineCode(option.Name))
-                .Append(": ");
-            builder.AppendLine(option.Display(cfg));
+        var footer = new StringBuilder();
+        const int optionsPerList = 7;
+        var totalPages = (AllOptions.Length + optionsPerList - 1)/optionsPerList;
+        for (var i = optionsPerList * page - optionsPerList; i <= optionsPerList * page - 1; i++) {
+            try {
+                builder.AppendLine($"Settings{AllOptions[i].Name}".Localized())
+                    .Append(Markdown.InlineCode(AllOptions[i].Name))
+                    .Append(": ")
+                    .AppendLine(AllOptions[i].Display(cfg))
+                    .AppendLine();
+            } catch { /* hilariously ignored */ }
```

I do not like this presence of try-catch without any explanation while also completely ignoring any exception that occurs.